src/solvers/refinement/string_constraint.h

```diff
@@ -114,6 +138,15 @@ extern inline string_constraintt &to_string_constraint(exprt &expr)
   return static_cast<string_constraintt &>(expr);
 }
 
+inline static std::string from_expr(
```

Comment:
Ah, it's used in about 4 or 5 places, mostly in `string_refinementt::is_valid_string_constraint`.